libcodechecker/libhandlers/server.py

```diff
@@ -342,6 +369,193 @@ def arg_match(options):
     parser.set_defaults(func=__handle)
 
 
+def handle_connect_ret(ret):
+    """
+    Print out a message based on the database statuses.
+    """
+    if ret == DBStatus.OK:
```

Comment:
You use this block of code in multiple places (https://github.com/gyorb/codechecker/blob/16b2583583258cf5a5935a01f3385b1c7ac9a388/libcodechecker/cmd/product_client.py#L46-L64). Create a helper function and use it in both places.